sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_04_with_parameters/cases/08_invalid_parameter.sql

```diff
@@ -0,0 +1,11 @@
+--+ server-message on
+-- verified the CBRD-24126
+
+create or replace procedure test_proc(a invalid_param) as
```

Comment:
It would be a good idea to also add types that are not supported.
And will you write a separate TC for javasp?

Reply:
javasp needs a *.java file.
Do you mean should I make the shell test case?

Reply:
I add the unsupported param test by javasp.
but invalid param test should be create by shell test.